tests/test_ctrlchannel

```diff
@@ -56,7 +56,7 @@ PID="$(cat $PID_FILE)"
 
 # Get the capability bits: CMD_GET_CAPABILITY = 0x00 00 00 01
 res="$(unix_tx '\x00\x00\x00\x01')"
-exp=" 00 00 00 00 00 00 07 ff"
+exp=" 00 00 00 00 00 00 17 ff"
```

Comment:
Are you sure this is 17 and not 0f ??

Reply:
For some reason, PTM_CAP_GET_CONFIG is missing in return ptm_caps, otherwise it would have been 1F FF, but now it is '17 FF'.

Reply:
I'll fix that then.

Reply:
@avalluri I fixed that now. Can you rebase?